libraries/AP_DAL/AP_DAL_GPS.cpp

```diff
@@ -3,6 +3,17 @@
 #include <AP_Logger/AP_Logger.h>
 #include "AP_DAL.h"
 
+#if HAL_GCS_ENABLED
```

Comment:
```suggestion
#if HAL_MAVLINK_BINDINGS_ENABLED
```
... this is, after all, where `GPS_FIX_TYPE_ON_GPS` are coming from